ts/editor/EditingArea.svelte

```diff
@@ -189,10 +194,25 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
 
         position: relative;
         background: var(--frame-bg);
-        border-radius: 0 0 5px 5px;
+        border-radius: 5px;
+        border: 1px solid var(--border);
+
+        box-shadow: 0px 0px 2px 0px var(--border);
+        transition: box-shadow 80ms cubic-bezier(0.33, 1, 0.68, 1);
 
-        &:focus {
+        &:focus-within {
             outline: none;
+            &::after {
+                content: "";
+                position: absolute;
```

Comment:
This also asks for a comment

Reply:
I had to delete that one to find out why it was required.

The highlight box-shadow is inset now (to not mess up alignment with field borders) and has to be displayed above the field contents. `PlainTextInput` covers the shadow when it's applied to `EditingArea` directly, so I went for a pseudo element.

Comment added in: https://github.com/ankitects/anki/pull/2002/commits/171cc063a22a7efbbeb5b34e2416816553fc1afc

Reply:
If there is a more elegant way to do it, I'm all ears. This hard-coded approach will need adjusting any time the gutters are changed in `Fields.svelte`, so it's far from ideal.